cmd/node/node.go

```diff
@@ -945,20 +945,20 @@ func (app *App) LoadOrCreateEdSigner() (*signing.EdSigner, error) {
 
 		log.Info("Identity file not found. Creating new identity...")
 
-		edSgn := signing.NewEdSigner(signing.WithSignerPrefix(app.Config.Genesis.GenesisID().Bytes()))
+		edSgn := signing.NewEdSigner(signing.WithPrefix(app.Config.Genesis.GenesisID().Bytes()))
 		err := os.MkdirAll(filepath.Dir(filename), 0o700)
 		if err != nil {
 			return nil, fmt.Errorf("failed to create directory for identity file: %w", err)
 		}
-		err = os.WriteFile(filename, edSgn.ToBuffer(), 0o600)
+		err = os.WriteFile(filename, edSgn.Bytes(), 0o600)
 		if err != nil {
 			return nil, fmt.Errorf("failed to write identity file: %w", err)
 		}
 
 		log.With().Info("created new identity", edSgn.PublicKey())
 		return edSgn, nil
 	}
-	edSgn, err := signing.NewEdSignerFromBuffer(data, signing.WithSignerPrefix(app.Config.Genesis.GenesisID().Bytes()))
+	edSgn, err := signing.NewEdSignerFromBuffer(data, signing.WithPrefix(app.Config.Genesis.GenesisID().Bytes()))
```

Comment:
unrelated change

Reply:
I find the name `WithSignerPrefix` confusing. The message is prefixed not the signer? Also the name is repetitive: `signing.WithSignerPrefix` contains the hint that it is signing twice.

Reply:
understood, the idea though is that there is more than one type in that module